sonic-net/sonic-mgmt
# [PCBB] Add testcase to verify PFC for tunnel traffic (Queue 2 and 6)

### Description of PR


Summary:
This PR is to add a test case to verify PFC for tunnel traffic.

Test steps:
1. Toggle all ports to active on randomly selected ports
2. Populate ARP table by GARP service
3. Disable Tx on egress port
4. Verify bounced back traffic (tunnel traffic, IPinIP) can trigger PFC at expected queue
5. Verify regular traffic can trigger PFC at expected queue

To verify PFC for tunnel traffic, the testing packets are ingressed from T1 (LAG). But all LAG members are removed in current `test_qos_sai.py`. To avoid making the current `testQosSaiPfcXoffLimit` too complicated, the new test case is implemented in a new module.

### Type of change





### Back port request

### Approach
#### What is the motivation for this PR?
This PR is to add a test case to verify PFC for tunnel traffic.

#### How did you do it?

#### How did you verify/test it?
Verified on `TD3` dualtor testbed. Verification on `TH2` is in progress.  

#### Any platform specific information?
No.

#### Supported testbed topology if it's a new test case?
Dualtor specific test case.

### Documentation
